plugin/src/test/scala/com/typesafe/genjavadoc/SignatureSpec.scala

```diff
@@ -132,6 +139,7 @@ class SignatureSpec extends WordSpec with Matchers {
         c.getDeclaredMethods.filterNot(x ⇒ filter && (defaultFilteredStrings.exists { s => x.getName.contains(s) }
           || javaKeywords.contains(x.getName)
           || x.getName == "$init$" // These synthetic methods show up in 2.12.0-M4+ even though they are not in the generated Java sources
+          || (filter && Modifier.isStatic(x.getModifiers) && x.getName.endsWith("$")) // These synthetic static methods appear since 2.12.0-M5+ as companions to default methods
```

Comment:
[Same approach](https://github.com/typesafehub/genjavadoc/commit/9705b7cec9c87f6841c4cda0b9e9b562280f2807) was taken when adding support for 2.12.0-M4. I think we can have it like this now, and when final scala 2.12.0 is out, we can filter on the output.